orca-web/src/test/groovy/com/netflix/spinnaker/orca/controllers/TaskControllerSpec.groovy

```diff
@@ -17,40 +17,50 @@
 package com.netflix.spinnaker.orca.controllers
 
 import java.time.Clock
+import java.time.Instant
+import java.time.ZoneId
+import groovy.json.JsonSlurper
 import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.orca.ExecutionStatus
 import com.netflix.spinnaker.orca.front50.Front50Service
+import com.netflix.spinnaker.orca.pipeline.PipelineStartTracker
 import com.netflix.spinnaker.orca.pipeline.model.*
 import com.netflix.spinnaker.orca.pipeline.persistence.ExecutionRepository
-import groovy.json.JsonSlurper
+import com.netflix.spinnaker.orca.restart.InstanceStatusProvider
+import org.springframework.http.HttpStatus
 import org.springframework.http.MediaType
 import org.springframework.mock.web.MockHttpServletResponse
 import org.springframework.test.web.servlet.MockMvc
 import org.springframework.test.web.servlet.setup.MockMvcBuilders
 import spock.lang.Specification
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.patch
+import static java.time.temporal.ChronoUnit.DAYS
+import static org.apache.commons.lang.RandomStringUtils.randomAlphanumeric
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*
 
 class TaskControllerSpec extends Specification {
 
   MockMvc mockMvc
-  ExecutionRepository executionRepository
-  Front50Service front50Service
+  def executionRepository = Mock(ExecutionRepository)
+  def front50Service = Mock(Front50Service)
+  def startTracker = Mock(PipelineStartTracker)
+  def instanceStatusProvider = Stub(InstanceStatusProvider)
 
-  Clock clock = Mock(Clock)
+  def clock = Clock.fixed(Instant.now(), ZoneId.systemDefault())
```

Comment:
This test was flaky on Travis CI so I just tidied it up — nothing to do with the main purpose of the PR.